ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -115,6 +115,21 @@ def test_numpy_ndarray_property_T(
         ground_truth_backend="numpy",
     )
 
+def test_numpy_ndarray_property_flat(
+    dtype_x,
+    item,
+):
+    dtype, data, shape = dtype_x
+    x = ndarray(shape, dtype[0])
+    x.ivy_array = data[0]
+    ivy.utils.assertions.check_equal(x.flat[item],data[0].flat[item],asarray=False)
+
+@given(
```

Comment:
A decorator should be right above a function definition, please move it to the line before `def test_numpy_ndarray_property_flat`